Tests/UnitTests/PlayMode/CardSets/UngoroCardsGenTests.cpp

```diff
@@ -364,6 +441,40 @@ TEST_CASE("[Neutral : Minion] - UNG_083 : Devilsaur Egg")
     CHECK_EQ(curField[0]->GetHealth(), 5);
 }
 
+// --------------------------------------- MINION - NEUTRAL
+// [UNG_806] Ultrasaur - COST:10 [ATK:7/HP:14]
+// - Race: Beast, Faction: Neutral, Set: Ungoro, Rarity: Common
+// --------------------------------------------------------
+TEST_CASE("[Neutral : Minion] - UNG_806 : Ultrasaur")
+{
+    GameConfig config;
+    config.player1Class = CardClass::HUNTER;
+    config.player2Class = CardClass::ROGUE;
+    config.startPlayer = PlayerType::PLAYER1;
+    config.doFillDecks = true;
+    config.autoRun = false;
+
+    Game game(config);
+    game.Start();
+    game.ProcessUntil(Step::MAIN_ACTION);
+
+    Player* curPlayer = game.GetCurrentPlayer();
+    Player* opPlayer = game.GetOpponentPlayer();
+    curPlayer->SetTotalMana(10);
+    curPlayer->SetUsedMana(0);
+    opPlayer->SetTotalMana(10);
+    opPlayer->SetUsedMana(0);
+
+    auto& curField = *(curPlayer->GetFieldZone());
+
+    const auto card1 =
+        Generic::DrawCard(curPlayer, Cards::FindCardByName("Ultrasaur"));
+
+    game.Process(curPlayer, PlayCardTask::Minion(card1));
+    CHECK_EQ(curField[0]->GetAttack(), 7);
+    CHECK_EQ(curField[0]->GetHealth(), 14);
+}
+
```

Comment:
This card don't need to testify because it has no any effects.
```suggestion
// --------------------------------------- MINION - NEUTRAL
// [UNG_806] Ultrasaur - COST:10 [ATK:7/HP:14]
// - Race: Beast, Faction: Neutral, Set: Ungoro, Rarity: Common
// --------------------------------------------------------
TEST_CASE("[Neutral : Minion] - UNG_806 : Ultrasaur")
{
    // Do nothing
}
```